docker/dev.docker-compose.yml

```diff
@@ -20,7 +20,7 @@ services:
     volumes:
       - ./cmd/run.sh:/run.sh
       - ../main:/main
-    command: ["bash", "/run.sh"]
+    command: ["sh", "/run.sh"]
```

Comment:
Just curious, any reason for this change?

Reply:
I don't think this image supports bash. I was unable to use docker exec -it <containerID> /bin/bash to enter the container but I can use /bin/sh

Reply:
By the way, why did I fail the travis CI check?

Reply:
Because the code is `gx-go rw`'ed, the import statements in your code are prefixed with "gx/ipfs/.../...", which cannot be recognized by `go get`